aws_cloudtrail_rules/aws_key_compromised.yml

```diff
@@ -0,0 +1,47 @@
+AnalysisType: rule
+Filename: aws_key_compromised.py
+RuleID: AWS.Key.Compromised
```

Comment:
Let's call this `AWS.IAM.AccessKeyCompromised`

Reply:
👍